Magento_GiftCard/templates/product/view/type/giftcard.phtml

```diff
@@ -52,11 +52,7 @@
                         type="email"
                         id="giftcard_sender_email"
                         name="giftcard_sender_email"
-                        class="
-                            validate-email
-                            required-entry
-                            input__field
-                        "
+                        class="validate-email required-entry input__field"
```

Comment:
Are these classes need to be unformatted?

Reply:
yep. otherwise, there's an issue with validating empty input. not sure why